lib/fog/ovirt/requests/compute/v4/list_vm_volumes.rb

```diff
@@ -3,18 +3,22 @@ module Compute
     class Ovirt
       class V4
         class Real
+          # rubocop:disable Metrics/AbcSize
           def list_vm_volumes(vm_id)
-            # disk_attachments_service = client.system_service.vms_service.vm_service(vm_id).disk_attachments_service
-            # disk_attachments_service.list.map {|ovirt_obj| ovirt_attrs client.system_service.disks_service.disk_service(ovirt_obj.id).get}
-
-            vm = client.system_service.vms_service.vm_service(vm_id).get
-
+             vm = client.system_service.vms_service.vm_service(vm_id).get
             attachments = client.follow_link(vm.disk_attachments)
-
             attachments.map do |attachment|
-              ovirt_attrs client.follow_link(attachment.disk)
+              attachment_disk = client.follow_link(attachment.disk)
+              at = client.follow_link(attachment)
```

Comment:
nitpick: could you change the name to `attachment`? I know it's a little confusing with `attachment_disk` but `at` confused me even more, I thought it was supposed to be `ad` for attachment disk at first.